learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -75,31 +75,31 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | Typo tolerant  | ✅ | ✅ | ✅ | 🔶 <br>Needs to be specified by fuzzy queries |
-| Orderable ranking rules | ✅ | ✅ | 🔶 <br>Tie-breaking order is limited by a unique scoring rule | ❌|
-| Custom rules | ✅ | ✅ | 🔶 <br>Limited to one default sorting rule | 🔶 <br>Function score query
+| Orderable ranking rules | ✅ | ✅ | 🔶 <br>Field priority can be changed, but ranking rules order cannot be changed. | ❌|
+| Custom rules | ✅ | ✅ | ✅ | 🔶 <br>Function score query
```

Comment:
Isn't Typesense limited to 3 `sort_by` rules max?

Reply:
That is only for hard-sorting. 

We handle ranking rules using query parameters which are not limited: 

https://typesense.org/docs/0.24.0/api/search.html#ranking-and-sorting-parameters
https://typesense.org/docs/0.24.0/api/search.html#typo-tolerance-parameters

Reply:
I'd like to change `Custom rules` to `Custom ranking rules` as I don't find explicit enough.
wdyt @gmourier ?

Reply:
Yes, it seems clearer!